modules/luci-mod-system/htdocs/luci-static/resources/view/system/flash.js

```diff
@@ -170,15 +170,17 @@ return view.extend({
 	},
 
 	handleBlock: function(hostname, ev) {
-		var mtdblock = dom.parent(ev.target, '.cbi-section').querySelector('[data-name="mtdselect"] select').value;
+		var mtdblock = dom.parent(ev.target, '.cbi-section').querySelector('[data-name="mtdselect"] select');
+		var mtdblockValue=mtdblock.value;
```

Comment:
Please adhere to the existing code style and put spaces around `=`. Also consider renaming the variables to reflect their meaning, e.g. `mtdnumber` and `mtdname`.

Reply:
Thanks @jow-. Added spaces and changed to `mtdnumber` and `mtdname` as requested.